src/power_grid_model/core/power_grid_model.py

```diff
@@ -334,6 +344,7 @@ def _calculate_short_circuit(
         threading: int = -1,
         output_component_types: Optional[Union[Set[str], List[str]]] = None,
         continue_on_batch_error: bool = False,
+        decode_error: bool = True,
         short_circuit_voltage_scaling: Union[ShortCircuitVoltageScaling, str] = ShortCircuitVoltageScaling.maximum,
```

Comment:
changing the order here is OK because of the `*` in line `341`. all arguments are kwargs